README.md

```diff
@@ -130,78 +130,20 @@ gem 'govuk_publishing_components', path: '../govuk_publishing_components'
 
-2. Drop and recreate any existing database, e.g. for Whitehall:
+Draft data can be replicated with `replicate-mongodb.sh draft-content-store` and `replicate-mongodb.sh draft-router`.
 
-```
-govuk-docker compose up -d mysql
-govuk-docker compose run mysql mysql -h mysql -u root --password=root -e "DROP DATABASE IF EXISTS whitehall_development"
-govuk-docker compose run mysql mysql -h mysql -u root --password=root -e "CREATE DATABASE whitehall_development"
-```
-
-3. Import the file into the local MySQL database, e.g. for Whitehall:
-
-```
-pv whitehall_production.dump.gz | gunzip | govuk-docker compose run mysql mysql -h mysql -u root --password=root whitehall_development
-```
-
-#### PostgreSQL
-
-1. Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/postgres/?region=eu-west-1&tab=overview)
-
-2. Drop and recreate any existing database, e.g. for Publishing API:
-
-```
-govuk-docker compose up -d postgres
-govuk-docker compose run postgres /usr/bin/psql -h postgres -U postgres -c DROP DATABASE IF EXISTS "publishing-api"
-govuk-docker compose run postgres /usr/bin/createdb -h postgres -U postgres publishing-api
-```
-
-3. Import the file into the local Postgres database, e.g. for Publishing API:
-
-```
-pv publishing_api_production.dump.gz  | gunzip | govuk-docker compose run postgres /usr/bin/psql -h postgres -U postgres -qAt -d publishing-api
-```
+Data is downloaded to `$GOVUK_DOCKER_REPLICATION_DIR`, which defaults to `$GOVUK_DOCKER_DIR/replication`.
 
-#### MongoDB
-
-1.  Download the relevant database dump from the [AWS S3 Bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups/mongodb/daily/mongo/?region=eu-west-1&tab=overview)
-
-2. Unzip the archive, e.g. for Content Store:
-
-```
-gunzip mongodump-2019-08-12_0023.tgz
-```
-
-Or if it's a TAR file, you can extract a specific file or directory.  Using the Content Store as an example:
-```
-tar -xvzf mongodump-2019-08-12_0023.tar var/lib/mongodb/backup/mongodump/content_store_production -C directory_for_download
-```
-
-3. Update the `docker-compose.yml` file to mount your local directory into the VM, e.g.
-
-```
-  mongo:
-    image: mongo:2.4
-    volumes:
-      - mongo:/data/db
-      - /Path/To/Downloads/directory_for_download:/import
-    ports:
-      - "27017:27017"
-      - "27018:27018"
-```
-
-4. Import the backup files into the local Mongo database, e.g. for Content Store:
-
-```
-govuk-docker compose up -d mongo
-govuk-docker compose run mongo mongorestore --drop --db content-store /import/var/lib/mongodb/backup/mongodump/content_store_production/
-```
+If the script hangs at "waiting for mongo..." (or whatever the database is), try stopping your govuk-docker containers and running the script again.  Data is cached, so it won't be downloaded a second time.
```

Comment:
Minor: it might be helpful (since I experienced it) to mention doing a `govuk-docker compose down` beforehand, since this can lock DBs. Perhaps this should be part of the scripts?